Fixed hashed passwords presented in KeePass app

ofersuli opened the pull request:
Fixed issue with exported kdbx files: entries' passwords were presented hashed when opening the kdbx file with KeePass app (version 2.38 for Windows OS)

ofersuli committed:
Update kdbx.cc

Fixed issue with exported kdbx files: entries' passwords presented hashed, when opened with KeePass app (version 2.38 for Windows OS)

ofersuli committed:
Merge pull request #1 from ofersuli/ofersuli-fix-export-hashed-passowrds

Fixed hashed passwords presented in KeePass

kindahl commented:
Great, many thanks for fixing this issue! May I ask that you fix the other instances in kdbx.cc that are also wrong? I believe all booleans serialized to XML should be serialized as "True" and "False". From what I can tell there are two more instances in kdbx.cc that needs fixing in a similar way to the one you fixed.

ofersuli committed:
kdbx.cc - fixed xml issue with true boolean value

Fixed issue -  true boolean value was written to the xml as "true" instead of "True".

ofersuli commented:
You're welcome!
You're right, I've fixed them too.
